packages/kernel-6.12/neuron-inf1.toml

```diff
@@ -0,0 +1,5 @@
+[neuron-inf1]
```

Comment:
This and the other `toml` should be `in.toml` as they are templated files, not the actual `toml` that will be used by the system